css-canvas-medical/styles.css

```diff
@@ -0,0 +1,270 @@
+header {
+  border-bottom: #f5f5f5;
+  border-bottom-style: solid;
+}
+
+a {
+  text-decoration: none;
+}
+
+body {
+  font-family: 'Avenir', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  color: #546163;
+}
+
+img {
+  width: 150px;
+}
+
+div > img {
+  width: 100%;
+  display: flex;
+  position: relative;
+  margin: 55px auto;
+}
+
+h1 {
+  position: absolute;
+  top: 142px;
+  left: 90px;
+  font-size: 2.3rem;
+}
+
+h3 {
+  margin-bottom: 5px;
+}
+
+li {
+  display: flex;
+}
+
+.container {
+  max-width: 1200px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.justify-evenly {
+  justify-content: space-evenly;
+}
+
+.justify-between {
+  justify-content: space-between;
+}
+
+.column-2 {
+  width: 55%;
+  margin: auto 40px;
+}
+
+.column-1 {
+  width: 15%;
+  margin: 20px 70px;
+}
+
+.column-3 {
+  width: 1050px;
+  margin: auto;
+}
+
+.text-center {
+  text-align: center;
+}
+
+.position {
+  position: relative;
+}
+
+.white {
+  color: white;
+}
+
+.button {
+  background-color: #163f64;
+  color: white;
+  border: none;
+  border-radius: 5px;
+  padding: 10px 15px;
+  cursor: pointer;
+}
+
+.button-2 {
+  background-color: #163f64;
+  color: white;
+  border: none;
+  border-radius: 5px;
+  padding: 18px 58px;
+  cursor: pointer;
+  font-size: 1rem;
+  position: absolute;
+  left: 90px;
+  top: 485px;
+}
+
+.p1 {
+  position: absolute;
+  top: 264px;
+  left: 90px;
+  width: 520px;
+  line-height: 24px;
+}
+
+.p3 {
+  line-height: 23px;
+  margin: 35px 300px 20px 65px;
```

Comment:
This is a lot of margin. I would look at using flex for this layout adjustment. If you put flex-direction: column on the column-6 element, you should be able to just justify content and align items to get the elements close to where you want them and then nudge them with margin.